Add sound related troubleshooting entries (#419)
jamulussoftware/jamulussoftware.github.io: wiki/en/en-Client-Troubleshooting.md

**Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4LL setup section](Installation-for-Windows#setting-up-asio4all)

### Getting a loud squeal if you connect to a server

> You're creating feedback since your microphone is recording the signal of your speaker. Disconnect from the server and select the correct input and output devices. See the installation page for your OS to get more information on how to do that.

Reviewer: "See the Jamulus installation page"? Otherwise it's a bit ambiguous as it could be referencing some installation page for the OS itself.

Author: Ok. added Jamulus before installation

> ### Not seing the headphone or microphone you just connected to your PC?

Reviewer: ```suggestion
### Not seeing the headphone or microphone you just connected to your PC?
```

> Jamulus currently doesn't show devices which were plugged in after the sound system was loaded. Just restart Jamulus with your device plugged in.

Reviewer: I'd swap these two sentences around, and consider not having the second. Non-engineers don't care why something is happening in this case, only how to fix it :-)

Author: Done.

### Your sound is stuttering

> Probably your audio device doesn't work with the buffer size you selected. Select higher buffer sizes in Jamulus's settings until you sound well. Use another device if possible since high buffer sizes mean high latency. 

Reviewer: ```suggestion
Your audio device may not work with the buffer size you selected. Select a bigger buffer size in Jamulus's settings. If possible though, use another device, since large buffer sizes mean higher latency. 
```
